pypit/arcomb.py

```diff
@@ -3,13 +3,40 @@
 import time
 
 import numpy as np
-from matplotlib import pyplot as plt
 
 from pypit import msgs
 from pypit import arparse as settings
-from pypit import arcycomb
 
-def comb_frames(frames_arr, det, frametype, weights=None, maskvalue=1048577, printtype=None):
+
+def comb_frames(frames_arr, det, frametype, **kwargs):
+    """ This method has been reduced to a simple wrapper to the core method.
+    It is likely to be deprecated in a future refactor
```

Comment:
Right, I think this should be deprecated.  Are you doing this at the moment just avoid having to alter the code that already calls `comb_frames()`?  I think with a function like this, we should already just go ahead and change the code so that you're calling `core_comb_frames()` (or just rename it to `comb_frames()`) directly instead of calling the wrapper.  I.e., anywhere you're currently calling `comb_frames`, I would instead call:

```
dnum = settings.get_dnum(det)
core_comb_frames(frames_arr, frametype=frametype,
                 method=settings.argflag[frametype]['combine']['method']
                 reject=settings.argflag[frametype]['combine']['reject'],
                 satpix=settings.argflag[frametype]['combine']['satpix'],
                 saturation=settings.spect[dnum]['saturation']*settings.spect[dnum]['nonlinear'])
```

Reply:
Am trying to avoid doing the refactoring in one fell swoop.
Especially since there are significant changes to settings underway.